### PR TITLE
Get coords from city name

### PR DESCRIPTION
**Pull request by rooneyshuman**
TODO: consider multiple cities with same name. Maybe also take state input?

* Converts city name to coordinates
* Reformats using "Prettier" file formatter
* Calls `get_tweets_and_sentiment()` on index page load

### CODE REVIEW

**Comment by rooneyshuman** [comment]
Will continue working on this. Currently fetches first matching city based on `city_name` input. Will need to expand to include country info or something to retrieve the correct city.

File: city_tweet_mood.js [hunk 1]
```diff
@@ -1,92 +1,96 @@
...
-function get_city_coordinates(city_name) {
-    // PLACEHOLDER FOR GETTING THE CITY COORDINATES VIA API
-    return '45.523,-122.676';
+function get_city_coordinates(city_name, state_name) {
+  let city = cities.find(
+    data => data.city.match(city_name) && data.state.match(state_name)
+  );
+  var lat = city.latitude.toString();
+  var lng = city.longitude.toString();
+  return lat + "," + lng;
```
Comment (logyball) [inline city_tweet_mood.js]: nitpicky: need to be vars? could probably just do `return city.latitude.toString() + ',' + city.longitude.toString()`

File: routes/index.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+  });
+});
+
+router.post("/mood", function(req, res) {
+  mood
+    .get_tweets_and_sentiment(req.body.city, req.body.state)
+    .then(tweets => {
+      res.render("tweets", { title: "TWITTER MOOD", tweets: tweets });
+    })
+    .catch();
```
Comment (logyball) [inline routes/index.js]: note to add error handling here - not necessary for this PR tho

File: views/index.ejs [hunk 3]
```diff
@@ -1,11 +1,101 @@
...
+              <option value="California">California</option>
+              <option value="Colorado">Colorado</option>
+              <option value="Connecticut">Connecticut</option>
+              <option value="Delaware">Delaware</option>
+              <option value="District of Columbia">District of Columbia</option>
+              <option value="Florida">Florida</option>
+              <option value="Georgia">Georgia</option>
+              <option value="Guam">Guam</option>
+              <option value="Hawaii">Hawaii</option>
+              <option value="Idaho">Idaho</option>
```
Comment (logyball) [inline views/index.ejs:44]: this hurts - any way we could load the options from the saved `cities.json`? That way we could have drop down menus for both options, then we wouldn't have to deal with free-text mismatches at all.  Not necessary for this PR but I think that would make using this a bit easier

**Review by logyball — APPROVED** [review]
This looks awesome.  One nitpicky code change and a question about the future

File: views/index.ejs (hunk 3 above)
Comment (rooneyshuman) [reply views/index.ejs:44]: Future PR: move the coord retrieval client-side

**Review by logyball — APPROVED** [review]

**Pull request merged**